test/fixtures/dialects/postgres/postgres_range_operators.yml

```diff
@@ -0,0 +1,178 @@
+# YML test files are auto-generated from SQL files and should not be edited by
+# hand. To help enforce this, the "hash" field in the file must match a hash
+# computed by SQLFluff when running the tests. Please run
+# `python test/generate_parse_fixture_yml.py`  to generate them after adding or
+# altering SQL files.
+_hash: 409b867a638598f0709224a6b3867912c5f4b0ef938648ff2a4618e37e408e53
+file:
+  statement:
+    select_statement:
+      select_clause:
+      - keyword: SELECT
+      - select_clause_element:
+          wildcard_expression:
+            wildcard_identifier:
+              identifier: word
+              dot: .
+              star: '*'
+      - comma: ','
+      - select_clause_element:
+          column_reference:
+          - identifier: paragraph
+          - dot: .
+          - identifier: id
+          alias_expression:
+            keyword: AS
+            identifier: paragraph_id
+      from_clause:
+        keyword: FROM
+        from_expression:
+          from_expression_element:
+            table_expression:
+              table_reference:
+                identifier: word
+          join_clause:
+          - keyword: INNER
+          - keyword: JOIN
+          - from_expression_element:
+              table_expression:
+                table_reference:
+                  identifier: paragraph
+          - join_on_condition:
+              keyword: 'ON'
+              expression:
+              - column_reference:
+                - identifier: paragraph
+                - dot: .
+                - identifier: page_id
+              - comparison_operator:
+                  raw_comparison_operator: '='
+              - column_reference:
+                - identifier: word
+                - dot: .
+                - identifier: page_id
+      where_clause:
+        keyword: WHERE
+        expression:
+        - column_reference:
+          - identifier: word
+          - dot: .
+          - identifier: character_range
+        - binary_operator: '@>'
+        - column_reference:
+          - identifier: paragraph
+          - dot: .
+          - identifier: character_range
+        - binary_operator: AND
+        - column_reference:
+          - identifier: word
+          - dot: .
+          - identifier: character_range
+        - binary_operator: <@
+        - column_reference:
+          - identifier: paragraph
+          - dot: .
+          - identifier: character_range
+        - binary_operator: AND
+        - column_reference:
+          - identifier: word
+          - dot: .
+          - identifier: character_range
+        - comparison_operator:
+          - ampersand: '&'
+          - ampersand: '&'
```

Comment:
We should try to parse these as one operator IMHO, which above changes should allow